modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/src/main/java/com/liferay/headless/commerce/admin/pricing/internal/resource/v1_0/DiscountAccountGroupResourceImpl.java

```diff
@@ -204,8 +204,12 @@ private List<DiscountAccountGroup> _toDiscountAccountGroups(
 	@Reference
 	private CommerceDiscountService _commerceDiscountService;
 
-	@Reference
-	private DiscountAccountGroupDTOConverter _discountAccountGroupDTOConverter;
+	@Reference(
+		target = "(component.name=com.liferay.headless.commerce.admin.pricing.internal.dto.v2_0.converter.DiscountAccountGroupDTOConverter)"
+	)
```

Comment:
Hey @RenanUmV, you used the wrong component.name 😢. Please break this commit in two, one for each DTO version. It'll avoid mistakes like this one. Thx

PS: please apply to other commits in this pr